docs/articles/expensify-classic/get-paid-back/expenses/Merge-Expenses.md

```diff
@@ -1,5 +1,62 @@
 ---
-title: Merge Expenses
-description: Merge Expenses
+title: Merge-Expenses.md
+description: This article shows you all the ways that you can merge your expenses in Expensify!
 ---
-## Resource Coming Soon!
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# About
+The merge expense function helps combine two separate expenses into one. This is useful when the same expense has been accidentally entered more than once, or if you have a connected credit card and an imported expense didn’t automatically merge with a manual entry. 
+
+# How-to merge expenses
+It’s important to note that merging expenses doesn't add the two values together. Instead, merging them combines both expenses to create a single, consolidated expense.
+
+Keep in mind: 
+1. Merging expenses cannot be undone.
+2. You can only merge two expenses at a time.
+3. You can merge a cash expense with a credit card expense, or two cash expenses - but not two credit card expenses.
+4. In order to merge, both expenses will need to be in an Open or Unreported state.
+# How to merge expenses on the web app
+To merge two expenses from the Expenses page:
+1. Sign into your Expensify account.
+2. Navigate to the Expenses page on the left-hand navigation.
+3. Click the checkboxes next to the two expenses you wish to merge.
+4. Click **Merge**.
+5. You'll be able to choose which aspect of each of the two expenses you would like to be used on the resulting expense, such as the receipt image, card, merchant, category, and more.  
+
+To merge two expenses from the Reports page:
+1. Sign into your Expensify account.
+2. Navigate to the Reports page on the left-hand navigation.
+3. Click the Report that contains the expenses that you wish to merge.
+4. Click on the **Details** tab, then the Pencil icon.
+5. Select the two expenses that you wish to merge.
+6. You'll be able to choose which aspect of each of the two expenses you would like to be used on the resulting expense, such as the receipt image, card, merchant, category, and more.   
+# How to merge expenses on the Expensify mobile app
+On the mobile app, merging is prompted when you see the message "_Potential duplicate expense detected_". Simply tap **Resolve Now** to take a closer look, then hit **Merge Expense**, and you're done! 
```

Comment:
```suggestion
On the mobile app, merging is prompted when you see the message _"Potential duplicate expense detected"_. Simply tap **Resolve Now** to take a closer look, then hit **Merge Expense**, and you're done! 
```

Reply:
technically the period should be inside the quotes, too. But that's a style thing.

Reply:
Oh yeah, lets change that - thanks!